evictionfree/tests/test_schema.py

```diff
@@ -91,9 +103,9 @@ def test_it_returns_error_when_only_phone_number_is_in_session(self):
         self.populate_phone_number()
         assert self.execute()["errors"] == self.INCOMPLETE_ERR
 
-    def test_it_returns_error_when_nyc_addr_but_onboarding_step_1_empty(self):
+    def test_it_returns_error_when_nyc_addr_but_legacy_onboarding_step_1_empty(self):
```

Comment:
ah, so legacy = onboardingstep1v2 and legacy_v1 == onboardingstep1?

Reply:
yeah i think so. um i wasn't suuper rigorous about the naming because it's gonna go away soon.